docs/guide/deployer.md

```diff
@@ -31,9 +31,12 @@ For opting in to ASA, `deployer` supports two methods :-
 - `optInAcountToASA` to opt-in to a single account signed by secret key of sender.
 - `optInLsigToASA` to opt-in to a contract account (say escrow) where the account is represented by the logic signature address (`lsig.address()`).
 
-#### Smart contracts
+### Smart contracts
 
-You can also deploy a Stateful Smart Contracts (SSC). Check our [examples/permissioned-voting](../examples/permissioned-voting) project. Open the `scripts/voting.js` file, you will find there:
+You can also deploy a Stateful/Stateless Smart Contracts (SSC).
+
+#### Stateful Smart Contracts
```

Comment:
Please add a link here to algorand developer docs for stateful contracts

Reply:
Okay